source/help/getting-started/creating-teams.rst

```diff
@@ -6,11 +6,17 @@ New teams can be created if the System Admin has **Enable Team Creation** set to
 Methods to Create a Team
 ------------------------
 
-Teams can be created from the Team Selection page or Main Menu. The person who creates a team will automatically be given the Team Admin role for that team.
+Teams can be created from the team sidebar or Main Menu. The person who creates a team will automatically be given the Team Admin role for that team.
```

Comment:
Maybe this should start with, `If team creation is enabled by the System Admin, ...`